nose/plugins/allmodules.py

```diff
@@ -21,6 +21,7 @@ def wantFile(self, file):
         """Override to return True for all files ending with .py"""
         if file.endswith('.py'):
             return True
+        return None
```

Comment:
Would this be better as:
```suggestion
        return file.endswith('.py')
```

Reply:
If it was `True` vs `False`, then it would be fine. But it's `True` vs `None`.

Reply:
Yes but given the function name, `def wantFile(self, file) --> bool:` seems more logical than
`def wantFile(self, file) --> bool | None:`

Reply:
https://github.com/mdmintz/pynose/blob/b6202befd6cecdf8ae2947fbabfbf741309a5861/nose/plugins/base.py#L560

Reply:
Probably doesn't make a difference. Technicalities.